<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>645</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>645</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>200</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Your Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DimLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dimensions:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dims">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>W: 0 H: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="percentDarkLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Occupancy %:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pDark">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="infolabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Image Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="numBlobsLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>150</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number of Blobs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nBlobs">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxBlobAreaLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max Blob Area:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBArea">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>75</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
